First request for landing

Hamuud opened the pull request.

StasLukianchuk reviewed (changes requested):
Great job, but your "Now on View" section doesn't work right on mobile

liliia-livandovskaya reviewed (changes requested):
Good job 🔥 Your landing looks great. There are some things to fix, please watch review https://www.loom.com/share/70ed4dbcb8594beeb1a21d08d00abec0

Hamuud commented:
Added a link to The Met website in the Museum HOURS section, animation for each museum separately, adaptation for a tablet, fixed the submission of the form, added color for all links, and added an appearance animation for the pseudo-element
